docs/source/transaction_family_specifications/integerkey_transaction_family.rst

```diff
@@ -0,0 +1,134 @@
+***************************************************
+IntegerKey Transaction Family Specification
+***************************************************
+
+Overview
+=========
+The IntegerKey transaction family allows users to set, increment, and decrement
+the value of entries stored in state. 
+
+An IntegerKey family transaction request is defined by the following values:
+
+* A verb which describes what action the transaction takes 
+* A name of the entry that is to be set or changed
+* A value by which the entry will be set or changed
+
+The 'set' verb is used to create new entries. The initial value of the entry will be 
+set to the value specified in the transaction request. The 'inc' and 'dec' verbs are 
+used to change the value of existing entries in state.
+
+State
+=====
+This section describes in detail how IntegerKey transaction information is stored 
+and addressed.
+
+The address values of IntegerKey transaction family entries are stored in state as
+a cbor encoded dictionary, with the key being the *Name* and the value being an integer 
+*Value*.
+
+.. code-block:: python
+
+    cbor({    
+        // The name of the entry : The current value of the entry
+        'Name':'Value'
+    })
+
+\ *Names* and *Values* must conform to the following rules:
+
+* Valid *Names* are utf-8 encoded strings with a maximum of 20 characters
+* Valid *Values* must be integers in the range of 0 through 2\ :sup:`32` − 1 (32-bit unsigned int)
+
+Addressing
+----------
+When an IntegerKey family transaction entry is stored, it can be accessed by addressing using 
```

Comment:
Maybe reword this to "IntegerKey data is stored in state using addresses which are generated from the intkey namespace prefix and the unique *Name* of the intkey variable. These addresses are generated using the following algorithm:"